Content.Server/SS220/Tarot/TarotColodeComponent.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Content.Server.SS220.Tarot;
+
+[RegisterComponent]
+public sealed partial class TarotColodeComponent : Component
+{
+    [DataField]
+    public List<string> CardsName = [];
```

Comment:
```suggestion
    [DataField]
    public List<ProtoId<EntityPrototype>> CardsProtos = [];
```

Если будешь на общий компонент переносить то думаю лучше использовать `EntityTableSelector`